polar-core/src/sugar.rs

```diff
@@ -219,8 +240,15 @@ impl Declaration {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub enum EntityType {
+    Actor,
+    Resource,
+}
+
 #[derive(Clone, Debug, PartialEq)]
 pub struct Namespace {
+    pub entity_type: EntityType,
     pub resource: Term,
```

Comment:
Renamed `entity_type` to `block_type` for now since the feature is just typed resource blocks. We'll be able to do away with `block_type`/`BlockType` once we have true union types.

Reply:
(also renamed `Namespace` -> `ResourceBlock`, so the `block_type` & `resource` fields make a bit more sense now)